framework/config/config.py

```diff
@@ -127,7 +127,10 @@ def ProcessOptionsPhase1(self, options):
 
     def ProcessOptionsPhase2(self, options):
         target_urls = self.LoadTargets(options)
-        self.load_works(target_urls, options)
+        if(options['UploadTool']):
```

Comment:
@DoomTaper wherever possible, please change the functions and variable names to be `var_name` type and classes to be `MyClass` case.